parsing/lexer.mll

```diff
@@ -36,12 +36,13 @@ type error =
   | Invalid_char_in_ident of Uchar.t
   | Non_lowercase_delimiter of string
   | Capitalized_raw_identifier of string
+  | Unknown_keyword of string
 
 exception Error of error * Location.t
 
 (* The table of keywords *)
 
-let keyword_table =
+let keyword_table_all =
   create_hashtable 149 [
```

Comment:
Could this be an association list, now that it isn't used in a performance-sensitive way anymore? It would help clarify what is mutable via configuration and what is not.